frontend/public/locales/en/apply.json

```
@@ -93,6 +93,31 @@
       "sun-life": "https://www.sunlife.ca/en/"
     }
   },
+  "tax-filing": {
+    "page-title": "Tax filing",
+    "form-instructions": "Have you and your spouse or common-law partner (if applicable) filed your tax return for 2023 and received your Notice of Assessment?",
```

From Figma

```suggestion
    "form-instructions": "Have you, and if applicable, your spouse or common-law partner, filed your tax return for {{taxYear}} in Canada and received your Notice of Assessment?",
```

Where `{{taxYear}}` will be dynamically rendered based on the `applicationYear.taxYear` field in the apply state.

(Same comment applies for French)